ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -473,6 +473,38 @@ def test_paddle_atanh(
     )
 
 
+# broadcast_shape
+@handle_frontend_test(
+    fn_tree="paddle.broadcast_shape",
+    dtype_and_x=helpers.dtype_and_values(
```

Comment:
Hey! Thanks for the changes! I've just come to realised the inputs to `broadcast_shape` expects shape integer or tuples, so I think this helper function is not the appropriate one, `helpers.get_shape` should be the one you're looking for. If necessary, you may generate data to 2 variables in the `handle_frontend_test` and pass them to the testing function.

Reply:
Sure, let me fix that now